src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1012,6 +1012,99 @@ def test_export_template_spec(self, resource_group, resource_group_location):
 
 
 
+class DeploymentTestsWithQueryString(LiveScenarioTest):
```

Comment:
May I ask why are they live only tests?

Reply:
The tests generate SAS tokens which fail the cred scan, so I have changed them to live tests.